portfolio/src/main/webapp/script.js

```diff
@@ -142,24 +142,34 @@ google.charts.load('current', {'packages':['corechart']});
 google.charts.setOnLoadCallback(drawChart);
 
 /** Creates a chart and adds it to the page. */
-/* TODO (Carolyn): Update this chart to my actual bubble chart */
 function drawChart() {
   const data = new google.visualization.DataTable();
-  data.addColumn('string', 'Animal');
-  data.addColumn('number', 'Count');
+  data.addColumn('string', 'Artist');
+  data.addColumn('number', 'Hours Played');
+  data.addColumn('number', 'Number of songs in my top 100 most played songs');
```

Comment:
because these strings are repeated below I think it would be good to create consts to hold them.

e.g. const number_of_songs = 'Number of songs in my top 100...'